### ISSUE TITLE
[Mac OS] 1.12.2 modpacks wont start

### ISSUE THREAD

**Issue by nothing6478** [FTBTeam/FTB-App#1173]
### What Operating System

MacOS (Latest)

### Debug Code

 FTB-DBGIPOPEHOPOK

### Describe the bug

After Modloading It it has crash all modpack 
Crash-Reports 
FTB present Stoneblock2
[crash-2024-07-23_11.54.31-client.txt](https://github.com/user-attachments/files/16343259/crash-2024-07-23_11.54.31-client.txt)
FTB presentDirewolf 20 1.12
[crash-2024-07-23_12.06.19-client.txt](https://github.com/user-attachments/files/16343264/crash-2024-07-23_12.06.19-client.txt)

### Steps to reproduce

Deleted App and all folder ex. .ftb
Re-Installed all (app, modpacks, Java)
But nothing has repaired

### Expected behaviour

I just want start the game

### Screenshots

![스크린샷 2024-07-23 오전 11 56 22](https://github.com/user-attachments/assets/4a3bc940-5364-4db4-92c8-5cd9dc7d68cd)

### Additional information

M2 MacbookAir / MacOs Sonoma 14.5

Before the bug I didn't modified any files and update Os, Java, App, Modpacks 
It worked Last sunday

**Comment by nothing6478** [comment]
*ps. I think the both modpacks have same contents of crash-report, but I can't understands that report, (why the crash occured)

**Comment by nothing6478** [comment]
*ps. I found out the cause through multiple searches. The reason is that it was recognized that there was no NaratorOSX Java classpath. However, the .jar file on the 
.ftb/bin/libraries/com/mojang/text2speach/1.10.3/text2speach-1.10.3.jar 
It was confirmed that NarratorOSX files were present in the jar. 

I think it's less likely that the file was damaged because I reinstalled it muliple times. I think Java doesn't seem to recognize that classpath. But I have a lot of difficulty solving that problem on my own because I don't know java very well.
So. That's all I've found out and I need your help.
